### DISCUSSION TITLE
Style Guide?

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#3696] [General]
What is the correct white spacer? Tab or 4 white spaces?

Where can I find the current style guide?

global.phtml:
![grafik](https://user-images.githubusercontent.com/1645099/125347435-35425200-e35b-11eb-95a0-89944606d169.png)

**Comment by aledeg** [comment]
If I am not wrong, it's described in the documentation.
The chosen style is a mix of both to my dismay

**Reply by Frenzie** [reply]
It's tabs, optionally with spaces for certain forms of alignment.
